src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -142,6 +142,101 @@ To host synthetics job managers, your system must meet the minimum requirements
     </Callout>
   </Collapser>
 
+  <Collapser
+    id="podman-requirements"
+    title={<><img src="/images/synthetic_logo_podman.webp" title="Podman icon" alt="Podman icon" style={{ height: '35px', width: '40px' }}/>Podman container system environment requirements</>}
+  >
+    <table>
+      <thead>
+      <tr>
+        <th style={{ width: "165px" }}>
+          Compatibility for
+        </th>
+
+        <th>
+          Requirements
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          Operating system
+        </td>
+
+        <td>
+          <DNT>**Linux kernel:**</DNT> 3.10 or higher<br/>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Processor
+        </td>
+
+        <td>
+          A modern, multi-core AMD64 or x86_64 CPU
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Memory
+        </td>
+
+        <td>
+          3.256 GiB of RAM per CPU core (dedicated)
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          Disk size
+        </td>
+
+        <td>
+          A minimum of 50 GiB per host
```

Comment:
why would we need 50GB?

Reply:
Memory requirement is same as that of the docker. This is consistent with what we have in our production servers as well.

Reply:
It is still unusual for us to ask for 50GB of disk space (i am hoping this is disk and not memory) but if this is whats expected for other deployment modes, then i guess its ok 🤷🏽‍♀️